cmd/nri-ecs.go

```diff
@@ -34,91 +34,59 @@ func main() {
 		os.Exit(1)
 	}
 
-	if args.DebugMode {
-		v3.DebugECSEndpoint()
```

Comment:
Do we know why this was added? Is it possible that in some scenario is still useful when triaging issues? 
maybe @alejandrodnm remembers something 😄

Reply:
That's just something to check the endpoints on startup, I think we copied it from an official AWS ECS client.